pastas/model.py

```diff
@@ -747,7 +751,7 @@ def solve(
         tmax: Optional[TimestampType] = None,
         freq: Optional[str] = None,
         warmup: Optional[float] = None,
-        noise: bool = True,
+        noise: bool = True,  # will be removed in version 2.0.0
```

Comment:
Same here, I think we should put this argument to None, and raise a deprecationwarning when used (either true or False), to inform users to no longer use this code and that to use a noise model, one needs to be added.